main.go

```diff
@@ -148,13 +146,28 @@ func main() {
 		app.autostartEnabled = true
 	}
 
-	hnsErrCh := make(chan error)
 	serverErrCh := make(chan error)
 	onBoardingFilename := path.Join(app.config.Path, "init")
 	onBoarded := onBoardingSeen(onBoardingFilename)
 
+	hnsdPath, err := getProcPath()
+	if err != nil {
+		log.Fatal(err)
+	}
+	hnsdPath = path.Join(hnsdPath, "/hnsd")
+
+	hnsdCheckpointPath := ""
+	if hnsdPath == "" {
+		log.Fatal("path to hnsd is not provided")
+	}
+	if hnsdCheckpointPath == "" {
+		home, _ := os.UserHomeDir() //above already fails if it doesn't exist
+		hnsdCheckpointPath = path.Join(home, ".hnsd")
```

Comment:
Instead of a new directory, we can probably use fingertip's existing one so all data stays in one place: `app.config.Path`

Reply:
Maybe.
Also maybe it's better to leave as is, because the user might also have hnsd, so both applications (fingertip and hnsd) will have share the checkpoint location.

Reply:
Not sure which is better: if user also has hnsd, then it's better to share the checkpoints location with hnsd.

Reply:
Oh forgot about this.If hnsd changes any file format in the future, it shouldn't affect fingertip. Same reason why Bob stores hsd data in its own directory, not in ~/.hsd.

It's also easier for the uninstaller to clean up, deleting one folder clears everything, and doesn't break any other hnsd that may still be installed independently.

And since we're using `-t`, even the first sync will be pretty fast anyway.